react-app/src/App.tsx

```diff
@@ -10,22 +10,9 @@ import { AuthProvider } from "./auth/AuthProvider";
 
 function App() {
   return (
-    <AuthProvider>
```

Comment:
?

Reply:
Ya the authprovider in app is still in both branches so it's fine